### PR TITLE
nomi edits

### PR DESCRIPTION
**Pull request by nslater**

### CODE REVIEW

File: blackbox/docs/ingestion.txt [hunk 1]
```diff
@@ -4,56 +4,43 @@
...
 
-:Ingestion Rule:
+How data flows into the data source is implementation specific. For example, a
+data source could get data from an MQTT_ endpoint, an HTTP webhook, or a module
+listening for Kernel events.
 
-  An ingestion rule defines where to put the incoming data of a
-  specific ingestion source, based on predefined conditions. For more,
-  details see :ref:`administration-ingestion-rules`
+CrateDB provides a number of builtin ingestion sources. More ingestion sources
```
Comment (andreidan) [inline blackbox/docs/ingestion.txt]: We just provide the `mqtt` one (so not "a number" but just the one), also the only plugin available is the mqtt ingestion implementation which we migrated to be part of crate (so will likely not be used/available anymore)

File: blackbox/docs/sql/administration/ingestion_rules.txt [hunk 2]
```diff
@@ -24,58 +24,55 @@
...
 
-In this section, we will learn about creating and deleting ingestion rules,
-listing the existing ingestion rules and finally the required privileges for
-the ``CREATE`` and ``DROP INGEST RULE`` statements, when the
-:ref:`administration_user_management` feature is enabled.
+This document will show you how to:
 
-Creating and Deleting Ingestion Rules
-.....................................
+- Create ingestion rules
```
Comment (andreidan) [inline blackbox/docs/sql/administration/ingestion_rules.txt]: should these be links?

File: blackbox/docs/sql/administration/ingestion_rules.txt [hunk 3]
```diff
@@ -24,58 +24,55 @@
...
+``mqtt_temperature`` table.
+
+We could do this::
 
-    cr> CREATE INGEST RULE temperature
+    cr> CREATE INGEST RULE mqtt_temperature_rule
     ... ON mqtt
     ... WHERE topic like 'temperature/%'
-    ... INTO temp_data.raw;
+    ... INTO mqtt_temperature.raw;
```
Comment (andreidan) [inline blackbox/docs/sql/administration/ingestion_rules.txt]: drop ".raw" to be consistent

File: blackbox/docs/ingestion.txt [hunk 4]
```diff
@@ -4,56 +4,43 @@
...
 ===================
 
-Aside from using standard SQL to ingest data, CrateDB offers the possibility to
-ingest data through custom event-based ingestion sources. The ingestion
-framework of CrateDB provides highly available and distributed ingestion
-sources for custom protocols where thousands of events are being sent
-simultaneously, such as MQTT_. With CrateDB's ingestion framework no
-intermediate queues or brokers need to be set up.
+As well as allowing you to ingest data via standard SQL inserts, CrateDB allows
+provides an ingestion framework that allows you to ingest data through custom
```
Comment (m-kharbat) [inline blackbox/docs/ingestion.txt]: allows / provides

File: blackbox/docs/sql/administration/ingestion_rules.txt (hunk 2 above)
Comment (m-kharbat) [reply blackbox/docs/sql/administration/ingestion_rules.txt]: i think it's not needed, the document is quite small

**Comment by nslater** [comment]
@andreidan @m-kharbat can you look at my fixup?

https://github.com/crate/crate/pull/6231/commits/980986049f36240a91a4d31ef5d55376bc5caa7b

basically, I've created a new section under our administration section, for collecting documentation about ingestion sources

I've added an mqtt document there, but it's empty. so it needs filling in?

sorry if I have misunderstood something!!

**Comment by nslater** [comment]
ignore my previous comment

**Pull request merged**